packages/clay-core/src/tree-view/TreeViewItem.tsx

```diff
@@ -86,6 +89,20 @@ export const TreeViewItem = React.forwardRef<HTMLDivElement, TreeViewItemProps>(
 								}
 							}
 						}}
+						onKeyDown={(event) => {
+							const {key} = event;
+							if (group) {
+								if (key === 'ArrowLeft') {
+									close(item.key);
+								}
+								if (key === 'ArrowRight') {
+									open(item.key);
+								}
+							}
+							if (key === 'Delete') {
```

Comment:
I'm not able to reproduce this case correctly, I'm not sure it could be because of the keyboard. The "Delete" key on the Macbook Pro keyboard, it's a Backspace too, maybe we should consider both?